tests/functional/test_six_threading.py

```diff
@@ -1,62 +0,0 @@
-"""
-Regression test for six issue #98 (https://github.com/benjaminp/six/issues/98)
```

Comment:
Just for my understanding - why are we removing the tests now as opposed to when we actually remove the module?  

Is there a benefit, or more of a "this has to be done sooner or later and since this is deprecated, the tests are of little value"

Reply:
Six hasn't been supported for several years and this is an issue for a significantly older version of six. It's no longer used in the library itself anymore so this test isn't relevant, that was all removed with Python 2.7 and the follow up work to clear six imports.